### PR TITLE
Bug 1179666 - Sort network names alphabetically within each signal ba…

### PR DESCRIPTION
**Pull request by scottwu**
…r level, r=gasolin

### CODE REVIEW

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=7c4ecbe32f20c9fe0c96de64100a796b06825f91

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f57869bd2169b0c33f6939c3103004ddac8e141

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=0f138935b6be78cea46ff35930066f6e48344e64

**Comment by scottwu** [comment]
Thanks Chris! I've edited the commit log.

**Comment by gasolin** [comment]
please add some test if possible

File: shared/js/wifi_helper.js [hunk 1]
```diff
@@ -249,5 +249,10 @@ var WifiHelper = {
       reqProxy.onerror();
     };
     return reqProxy;
+  },
+
+  // Returns signal strength in terms of signal bars
+  getSignalBars: function(network) {
```
Comment (gasolin) [inline shared/js/wifi_helper.js]: how about `getSignalLevel`?

**Comment by autolander** [comment]
Running tests
https://treeherder.mozilla.org/#/jobs?repo=gaia&revision=600578c7412677a8fd58224210507a8d65e6dc89

**Pull request merged**